cli/tests/test_root.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::path::Path;
+
+use crate::common::TestEnvironment;
+
+pub mod common;
+
+#[test]
+fn test_root() {
```

Comment:
Use the `testcase` to split the test. 

See https://github.com/martinvonz/jj/blob/6caeacf29e60a2957f81a25fe0cfcddccd887fed/lib/tests/test_local_working_copy.rs#L68-L71 as an example.

Reply:
I did that. It's a little bit weird that the `TestEnvironment::default()` (for calling the jj exe) and `TestWorkspace::init_with_backend` (for creating the repos) don't work together (they both use their own tmpdir), but it's minor.